src/identifiers.md

```diff
@@ -60,4 +63,37 @@ See the [HIR chapter][hir-map] for more detailed information.
 
 ## In the MIR
 
-**TODO**
+- [`BasicBlock`] identifies a *basic block*. It points to an instance of
+  [`BasicBlockData`].
+
+- [`Local`] identifies a local variable in a function. Its associated data is in
+  [`LocalDecl`].
```

Comment:
Same here?
```suggestion
  [`LocalDecl`] which can be retrieved with [`Body::local_decls`].
```

Reply:
It can be retrived by indexing `Body::local_decls`